### PR TITLE
[zh] translate all core-concepts docs.

### PR DESCRIPTION
**Pull request by SAMZONG**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

translate translate all core-concepts docs.

- i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/introduction.md
- i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/architecture.md

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/introduction.md [hunk 1]
```diff
@@ -1,54 +1,51 @@
...
-## Why Karmada:
-- __K8s Native API Compatible__
-    - Zero change upgrade, from single-cluster to multi-cluster
-    - Seamless integration of existing K8s tool chain
+## 为什么选择 Karmada
 
-- __Out of the Box__
-    - Built-in policy sets for scenarios, including: Active-active, Remote DR, Geo Redundant, etc.
-    - Cross-cluster applications auto-scaling, failover and load-balancing on multi-cluster.
+- __兼容 K8s 本机 API__
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/introduction.md:17]:
```suggestion
- __兼容 K8s 原生 API__
```

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/introduction.md [hunk 2]
```diff
@@ -1,54 +1,51 @@
 ---
-title: What is Karmada?
+title: Karmada 是什么？
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/introduction.md:2]: 这篇以前的中文哪儿去了....

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/introduction.md [hunk 3]
```diff
@@ -1,54 +1,51 @@
...
 slug: /
 
 ---
 
-## Karmada: Open, Multi-Cloud, Multi-Cluster Kubernetes Orchestration
+## Karmada：开放的、多云的、多集群的 Kubernetes 编排
 
-Karmada (Kubernetes Armada) is a Kubernetes management system that enables you to run your cloud-native applications across multiple Kubernetes clusters and clouds, with no changes to your applications. By speaking Kubernetes-native APIs and providing advanced scheduling capabilities, Karmada enables truly open, multi-cloud Kubernetes.
+Karmada（Kubernetes Armada）是一个 Kubernetes 管理系统，使您能够在多个 Kubernetes 集群和云中运行云原生应用程序，而无需更改应用程序。通过讲述 Kubernetes 本机 API 并提供先进的调度功能，Karmada 实现了真正的开放式、多云 Kubernetes。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/introduction.md:9]:
```suggestion
Karmada（Kubernetes Armada）是一个 Kubernetes 管理系统，使您能够在多个 Kubernetes 集群和多云中运行云原生应用，而无需更改应用。通过讲述 Kubernetes 原生 API 并提供先进的调度功能，Karmada 实现了真正的开放式、多云 Kubernetes。
```

**Review by windsonsea — COMMENTED** [review]
introduction.md 以前有中文的，怎么能找回呢

一个 term：
- application 应用

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/introduction.md (hunk 3 above)
Comment (Poor12) [inline i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/introduction.md:9]:
```suggestion
Karmada（Kubernetes Armada）是一个 Kubernetes 管理系统，使您能够在多个 Kubernetes 集群和云中运行云原生应用程序，而无需更改应用程序。通过使用 Kubernetes 原生 API 并提供先进的调度功能，Karmada 实现了真正的开放式、多云 Kubernetes。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/introduction.md (hunk 2 above)
Comment (SAMZONG) [reply i18n/zh/docusaurus-plugin-content-docs/current/core-concepts/introduction.md:2]: i dont know ...

**Comment by Poor12** [comment]
Please rebase two commits to one.
/lgtm

**Pull request merged**